src/Pokemon.java

```diff
@@ -0,0 +1,68 @@
+public abstract class Pokemon {
+
+    private String name;
+    private int level;
+    private int hp;
+    private String food;
+    private String sound;
+    private String type;
```

Comment:
In principe kunnen alle variabelen, behalve hp, ook `final` worden. Ze kunen dan geen setter meer hebben. Dat hoeft ook niet, want ze worden nooit aangepast.